src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -597,4 +599,21 @@ public function removeParameter($parameterName)
             }
         }
     }
+
+    /**
+     * Fetch an array of all return statements found within this function.
```

Comment:
@asgrim before we merge this: do you know if this code works also against the closure in the example above? Basically, if I reflect the closure above, will I be able to find `        return endTheWorld(); // this return statement "belongs" to the closure inside "foo"` ?